### ISSUE TITLE
For layers (or models) have an option for boolean or string parameter values

### ISSUE THREAD

**Issue by hassannaveed1997** [DSGT-DLP/Deep-Learning-Playground#453]
Lets say we have a layer, such as [Conv2d](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html) with a padding parameter. Now padding can be a number, but it can also be 'same', which is very useful when we don't want the convolutions to affect the size of the output. This problem occurs in many other models or layers.

Similarly, some parameters might be boolean. For example, in a Regression we might want to choose whether or not we want an intercept. It would be better to have that built in as a boolean.
